files/image_config/hostcfgd/hostcfgd

```diff
@@ -44,8 +44,6 @@ class AaaCfg(object):
     def __init__(self):
         self.auth_default = {
             'login': 'local',
-            'failthrough': True,
-            'fallback': True
```

Comment:
why change fallback as well? is fallback same as failthrough?

Reply:
No.
fallback is an unused variable. Just house cleaning.